bedrock/base/management/commands/get_geoip_db.py

```diff
@@ -0,0 +1,25 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+import requests
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        db_url = settings.MAXMIND_DB_URL
+        if db_url is None:
```

Comment:
nit: I'd go with the more generic `if db_url` just in case db_url is set to empty string for example.